chaos_monkey/network/network_scanner.py

```diff
@@ -27,48 +52,72 @@ def initialize(self):
         LOG.info("Found local IP addresses of the machine: %r", self._ip_addresses)
         # for fixed range, only scan once.
         if WormConfiguration.range_class is FixedRange:
-            self._ranges = [WormConfiguration.range_class(None)]
+            self._ranges = [WormConfiguration.range_class(fixed_addresses=WormConfiguration.range_fixed)]
```

Comment:
I think if we're doing this, we should remove the _config member from NetworkRange, stop allowing default "fixed_addresses" parameter on FixedRange c'tor, and remove the part where if fixed_addresses=None, we take it from config

Reply:
Fixed